app/pages/settings/email.jsx

```diff
@@ -292,12 +313,27 @@ class EmailSettingsPage extends React.Component {
                   <i className='fa fa-times-circle' style={{ color: '#e35950' }} />
                   {' '}
                   <Translate content="emailSettings.general.emailUnverified" />
-                  {/*
                   {' | '}
-                  <a href="#" onClick={this.requestConfirmationEmail}>
-                    <Translate content="emailSettings.general.emailUnverifiedPrompt" />
-                  </a>
-                  */}
+                  {(this.state.requestConfirmationEmailStatus === 'idle') && (
+                    <a href="#" onClick={this.requestConfirmationEmail}>
```

Comment:
This link links back to the current page. I think you meant to use a button here. In a screen reader, or even as a sighted user, I would expect the browser to navigate to a new location after clicking a link. I think the screen reader is more confusing, as this will be announced as a link.

Reply:
See the [WebAIM accessibility cheat sheet](https://webaim.org/resources/htmlcheatsheet/) for more info about accessible links, but the code here would fail an accessibility audit.